services/skus/datastore.go

```diff
@@ -1212,7 +1212,21 @@ func (pg *Postgres) InsertTimeLimitedV2OrderCredsTx(ctx context.Context, tx *sql
 		return fmt.Errorf("error marshaling signed creds: %w", err)
 	}
 
-	query := `insert into time_limited_v2_order_creds(item_id, order_id, issuer_id, blinded_creds,
+	// is this validity time past the order's expiry?  if so don't insert
+	ok := false
```

Comment:
I am wondering if this would be better as part of the [inserts signed order creds](https://github.com/brave-intl/bat-go/blob/5c277afe70b4a1107d89249b3f0d1da22bd23cc4/services/skus/datastore.go#L1406) e.g. pull back the order and check the dates or have a separate db call doing the check in the inserts signed order creds. This would keep out db funcs as simple crud ops